homeassistant/components/mqtt/__init__.py

```diff
@@ -585,6 +584,53 @@ class Message:
 class MQTT:
     """Home Assistant MQTT client."""
 
+    class AsyncioHelper:
```

Comment:
We shouldn't nest this inside the `MQTT` class. Flat is better than nested.

Reply:
OK, i got rid of the `AsyncioHelper` class, it was a bit redundant.